pox-locking/src/pox_2.rs

```diff
@@ -370,7 +368,7 @@ fn handle_stack_lockup_extension_pox_v2(
         }
     };
 
-    match pox_lock_extend_v2(&mut global_context.database, &stacker, unlock_height as u64) {
+    match pox_lock_extend_v2(&mut global_context.database, &stacker, unlock_height) {
```

Comment:
I'm not sure how I feel about dropping the `return` statements in these files.  I know it's what clippy wants, but because these return values are present in the middle of the code body, it's easier on the reader to see an explicit `return` to know that this is where flow-control stops.

Reply:
I can certainly silence the warnings instead. Let me go through and find ones that happen in what appears to be middle of code paths/where its non obvious and revert.